src/shared-components/alert/test.tsx

```diff
@@ -1,65 +1,62 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import TestRenderer from 'react-test-renderer';
-import { shallow } from 'enzyme';
 
 import Alert from './index';
 
-const testAlert = {
-  text: 'Your email address was updated successfully!',
-  type: 'success',
-  duration: 'sticky',
-};
+// Note on truncateText prop test: this cannot be tested because element scrollHeight and offsetHeight are not simulated correctly
+
+const alertText =
+  'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ex quis dicta iusto a placeat!';
 
-const CustomContentComponent = () => (
+const CustomContentComponent = (): JSX.Element => (
   <div>
     <strong>Your info is updated!</strong>{' '}
-    <a href="http://google.com"> Click here to see changes</a>
+    <a href="curology.com"> Click here to see changes</a>
   </div>
 );
 
-const createNodeMock = element => {
+const createNodeMock = (element: ReactElement): any => {
```

Comment:
```suggestion
const createNodeMock = (element: ReactElement): { innerHTML: string } | null => {
```

(I'm also comfortable just leaving returns like this not explicitly typed.)